lib/src/layer/polygon_layer/polygon_layer.dart

```diff
@@ -90,6 +105,21 @@ class _PolygonLayerState<R extends Object> extends State<PolygonLayer<R>>
     with
         ProjectionSimplificationManagement<_ProjectedPolygon<R>, Polygon<R>,
             PolygonLayer<R>> {
+  @override
+  void initState() {
+    if (kDebugMode && kIsWeb && widget.invertedFill != null) {
+      Logger(printer: PrettyPrinter(methodCount: 0)).w(
+        '\x1B[1m\x1B[3mflutter_map\x1B[0m\nOn the web, inverted filling may '
+        'not work as expected in some cases. It will not match the behaviour\n'
+        'seen on native platforms.\nAvoid allowing polygons to intersect, and '
+        'avoid using holes within polygons.\nThis is due to multiple '
+        'limitations/bugs within Flutter. See online documentation for more '
+        'info.',
```

Comment:
If possible, link the online resources like https://github.com/flutter/flutter/issues/124675

Reply:
This is linked from our online docs: https://docs.fleaflet.dev/layers/polygon-layer#inverted-filling